subsys/mgmt/mcumgr/lib/cmd/fs_mgmt/include/fs_mgmt/fs_mgmt.h

```diff
@@ -14,7 +14,9 @@ extern "C" {
 /**
  * Command IDs for file system management group.
  */
-#define FS_MGMT_ID_FILE	 0
+#define FS_MGMT_ID_FILE		0
+#define FS_MGMT_ID_CRC32	1
+#define FS_MGMT_ID_SHA256	2
```

Comment:
Do not add these as separate commands, use one command like `FS_MGMT_ID_CHKSUM` instead with param that would allow to select checksum algorithm. Otherwise we start to bloat IDs with all the requested checksums.